vividus/src/main/java/org/vividus/softassert/JBehaveFailTestFastManager.java

```diff
@@ -21,26 +21,44 @@
 import org.vividus.batch.BatchConfiguration;
 import org.vividus.batch.BatchStorage;
 import org.vividus.context.RunContext;
+import org.vividus.testcontext.TestContext;
 
 public class JBehaveFailTestFastManager implements FailTestFastManager
 {
+    private static final String FAIL_TEST_CASE_FAST_KEY = "FailTestCaseFast";
```

Comment:
it's better to use some unique key to be sure that no one else will use the same key, general rule is:
```suggestion
    private static final Object FAIL_TEST_CASE_FAST_KEY = JBehaveFailTestFastManager.class;
```